cypress/helpers/repository.js

```diff
@@ -11,7 +11,10 @@ export function pushLocalChanges(numberOfChanges) {
 
   if (numberOfChanges !== undefined) {
     // checking number of changes
-    menu.changesNumber().contains(numberOfChanges).should("be.visible");
+    menu
+      .changesNumber()
+      // .contains(numberOfChanges) <-- see this... it doesn't work 🙀
```

Comment:
Worked (or appeared to) until now. What triggered this?

Reply:
Retries, when it fails and retires the number of changes was 2